index.ts

```diff
@@ -55,7 +55,7 @@ async function main() {
             await exec("git", ["show", commit_hash]);
         }
         // Fetch and check out PR head
-        await exec("git", ["fetch", "--depth=1", "origin", `+refs/pull/${event.pull_request.number}/head`]);
+        await exec("git", ["fetch", "--depth=1", "origin", event.pull_request.head.ref]);
```

Comment:
I found that probably this doesn't work in case of pull requests from fork repositories.

Reply:
Checkout `event.pull_request.head.sha` instead of `event.pull_request.head.ref`.
https://github.com/actions/checkout?tab=readme-ov-file#Checkout-pull-request-HEAD-commit-instead-of-merge-commit